### PR TITLE
Allow global render() function to pass data and turn off auto sanitizatio

### PR DESCRIPTION
**Pull request by phil-lavin**
Allow global render() function to pass data and turn off auto sanitization

### CODE REVIEW

File: base.php [hunk 1]
```diff
@@ -145,13 +145,14 @@ if ( ! function_exists('get_object_public_vars'))
  *
  * @param	string	The view name/path
  * @param	array	The data for the view
+ * @param       bool    Auto sanitize data
  * @return	string
  */
 if ( ! function_exists('render'))
 {
-	function render($view, $data = array())
+	function render($view, $data = array(), $auto_sanitize = true)
```
Comment (frankdejonge) [inline base.php]: The default values should be the same as those from View::forge, both should be null.

File: base.php [hunk 2]
```diff
@@ -145,13 +145,14 @@ if ( ! function_exists('get_object_public_vars'))
...
 	{
-		return \View::forge($view, $data)->render();
+		return \View::forge($view, $data, $auto_sanitize)->render();
```
Comment (frankdejonge) [inline base.php]: Could you rename $auto_sanitize to $auto_filter to keep it inline with the other param naming?

**Comment by phil-lavin** [comment]
That ok?

**Comment by frankdejonge** [comment]
Merged, made $data default to null too (same as in View::forge).
Thanks!

**Pull request merged**